src/app/app.scss

```diff
@@ -1 +1,11 @@
-@import 'assets/nhsuk-8.1.1.min.css'
+@import 'assets/nhsuk-8.1.1.min.css';
+
+html {
+  font-size: 100%;
+  background-color: #fff;
+}
+
+body {
+  background: #fff;
+  overflow-x: hidden;
```

Comment:
Why do we have an overflow problem?

Reply:
We don't, and this is to prevent us from having it. It's the standard in CSS

Reply:
This would hide any potential problems in the future. I'd rather see the overflow than something that is potentially cut off.

I think we something does have an overflow, it would be better to fix in the component rather than globally? 

Just reading about this as a practice I'm seeing more "it's bad" than "good".